lib/f_service/result/base.rb

```diff
@@ -87,9 +100,9 @@ def on_success(*target_types)
       #   class UsersController < BaseController
       #     def update
       #       User::Update.(user: user)
-      #                   .on_success { |value| return json_success(value) } # this won't run
+      #                   .on_success(:unhandled: true) { |value| return json_success(value) } # this won't run
```

Comment:
Same here, I don't think using `unhandled: true` should be the default example. Let's keep it simple, since this is a more specific use case

Reply:
Well, as I've mentioned [here](https://github.com/Fretadao/f_service/pull/30#discussion_r762937130) seems to not have sense to keep `.on_success`, `.on_failure`, `.on_success(unhandled: true)` and `.on_failure(unhandled: true)` living together since they do exactly same thing...

Then since we decide to have unhandled arg to process any result passing by that callback, doesn't make sense to let the older one living...